docs/participant.md

```diff
@@ -279,6 +280,73 @@ Note: This is only allowed with validate SIP bridge requests
 
     - Data: See array definition in `Get user´s conversations`
 
+## Verify a dial-in PIN
+
+Note: This is only allowed as validate SIP bridge requests
+
+* Required capability: `sip-support-dialout`
+* Method: `POST`
+* Endpoint: `/room/{token}/verify-dialin`
+* Data:
+
+| field | type   | Description                         |
+|-------|--------|-------------------------------------|
+| `pin` | string | PIN the participant used to dial-in |
+
+* Response:
+    - Status code:
+        + `200 OK`
+        + `401 Unauthorized` When the validation as SIP bridge failed
+        + `404 Not Found` When the conversation or participant could not be found
+        + `501 Not Implemented` When SIP is not configured
+
+    - Data: See array definition in `Get user´s conversations`
+
+## Verify a dial-out number
+
+Note: This is only allowed as validate SIP bridge requests
+
+* Required capability: `sip-support-dialout`
+* Method: `POST`
+* Endpoint: `/room/{token}/verify-dialout`
+* Data:
+
+| field     | type   | Description                                                                                                                     |
+|-----------|--------|---------------------------------------------------------------------------------------------------------------------------------|
+| `number`  | string | E164 formatted phone number                                                                                                     |
+| `options` | string | Additional details to verify the validity of the request as JSON encoded object (Should contain actorId, actorType, attendeeId) |
+
+* Response:
+    - Status code:
+        + `200 OK`
+        + `401 Unauthorized` When the validation as SIP bridge failed
+        + `404 Not Found` When the conversation or participant could not be found
+        + `501 Not Implemented` When SIP or SIP dial-out is not configured
+
+    - Data: See array definition in `Get user´s conversations`
+
+## Reset call ID of rejected dial-out
+
+Note: This is only allowed as validate SIP bridge requests
+
+* Required capability: `sip-support-dialout`
+* Method: `DELETE`
+* Endpoint: `/room/{token}/rejected-dialout`
+* Data:
+
+| field        | type   | Description                                 |
+|--------------|--------|---------------------------------------------|
+| `attendeeId` | int    | The attendee ID of the dial-out participant |
```

Comment:
In order to not having to parse the `options` to extract the `attendeeId`, I would rather pass the `options` as-is just like it is done for `verify-dialout`.

```suggestion
| `options`    | string | The options as received in the dialout request. |
```

Reply:
Makes sense I guess